site/about/overview-llm-features.qmd

```diff
@@ -138,4 +138,6 @@ Discover how {{< var vm.product >}}’s LLM-powered platform, purpose-built for
 
 <!-- FOOTNOTES -->
 
-[^1]: [Run tests and test suites](//developer/model-testing/testing-overview.qmd)
\ No newline at end of file
+[^1]: [Run tests and test suites](//developer/model-testing/testing-overview.qmd)
+
+[^2]: [AI Usage Policy](https://validmind.com/about/legal/ai-usage-policy/)
```

Comment:
If you have the direct link then this footnote isn't needed — can we pick one or the other?

```
[WARNING] Note with key '2' defined at line 122 column 1 but not used.
```

Reply:
I already removed this in a commit not too long ago.

Reply:
Yup, removed. The remaining footnote is now commented out but might remove it as well.